src/pyenphase/envoy.py

```diff
@@ -233,6 +238,19 @@ async def _request(
                 f"Authentication failed for {url} with status {status_code}, "
                 "please check your username/password or token."
             )
+        # show all responses centrally when in debug
+        if debugon:
+            content_type = response.headers.get("content-type")
+            content = response.content
+            if content_type == "application/json":
+                content = orjson.loads(content)
```

Comment:
Why parse the json here? It might be better to log it in case it's invalid or otherwise has an error that will be less obvious if it's mutated with loads

Reply:
Only to make it a bit better readable. But you are right. I'll change to just dump content. That's what I wanted in the end :-)

Reply:
Changed